### ISSUE TITLE
Component styles missing on error pages

### ISSUE THREAD

**Issue by MaxMilton** [sveltejs/sapper#303]
Since Sapper `v0.14.0` component styles are missing on error pages. I assume it's related to <https://github.com/sveltejs/sapper/pull/300>.

**Comment by Rich-Harris** [comment]
I'm not seeing this... can you elaborate?

**Comment by MaxMilton** [comment]
This happens when there are styles in `App.html` or a component used in `App.html`. Normally a style block is added to the head when the route is loaded... but this doesn't happen on error pages.

I'm going to wait until https://github.com/sveltejs/sapper/pull/308 drops since it may inadvertently fix this. If it's still an issue afterwards then I'll create a minimal repro.

**Comment by Rich-Harris** [comment]
0.15 is out — if you get a chance to see if it's fixed now, let me know! thanks

**Comment by MaxMilton** [comment]
After a bit of debugging, I figured it out! I'm using [sapper-template](https://github.com/sveltejs/sapper-template) and in my `server.config.js` I had a Svelte style preprocessor which returned nothing.

It didn't occur to me that the SSR styles were used to generate `<style>` blocks in the head. Once I set up style preprocessing the same as `client.config.js` then things are now working as expected.

Sorry for any confusion!
